.bazeliskrc

```diff
@@ -1,2 +1,2 @@
 # See https://github.com/bazelbuild/bazelisk
-USE_BAZEL_VERSION=6.0.0
```

Comment:
BTW, we are still using 6.0.0 in [googleapis](https://github.com/googleapis/googleapis/blob/288aa7fb71c9b6244db1a816cfd4108f811e6049/.bazeliskrc#L2), so it's better to keep in sync with it.

Reply:
Reverted.